src/util/bundleUtils.js

```diff
@@ -146,8 +146,12 @@ async function getDependentValueSets(lib) {
   }
 
   const depValueSetUrls = lib.relatedArtifact
-    .filter(ra => ra.type === 'depends-on' && ra.resource.includes('ValueSet'))
-    .map(ra => ra.resource);
+    .filter(
+      ra =>
+        ra.type === 'depends-on' &&
+        ((ra.resource && ra.resource.includes('ValueSet')) || (ra.url && ra.url.includes('ValueSet')))
```

Comment:
The use of `url` for the ValueSet references is incorrect. Remove this change.

Reply:
Removed the check on `url` and switched to using a `?` to make sure `resource` exists.